#error and ok
int8 PROCESS_OK=0
int8 PROCESS_ERROR=-1


#const for object detecting
int8 OBJ_DETECT_CHARGE=1


#const for recording
int8 RECORD_START=1
int8 RECORD_STOP=2
int8 RECORD_ERROR=3



#const for p2p av module
int8 P2P_AV_PLAYING=1
int8 P2P_AV_STOP=2
int8 P2P_AV_ERROR=3

#const define for wifi
int8 WIFI_MODE_AP=0
int8 WIFI_MODE_STA=1

int8 WIFI_STATUS_DISCONNECT=0
int8 WIFI_STATUS_CONNECTED=1
int8 WIFI_STATUS_CONNECTING=2
int8 WIFI_STATUS_WRONG_KEY=3
int8 WIFI_STATUS_CONN_FAIL=4
int8 WIFI_STATUS_STOP=5

int8 BACK_UP_DETECT=1
int8 BACK_UP_ALIGN=2
int8 BACK_UP_BACK=3
int8 BACK_UP_SUCCESS=4
int8 BACK_UP_FAIL=5
int8 BACK_UP_INACTIVE=6
int8 BACK_UP_CANCEL=7
int8 BACK_UP_REDETECT=8

int8 BATTERY_CHARGING=0
int8 BATTERY_UNCHARGE=1
int8 BATTERY_FULL=2
int8 BATTERY_UNKOWN=3

#msg define
int32[]  status